lib/Cargo.toml

```diff
@@ -12,7 +12,7 @@ documentation = "https://docs.rs/isomdl-18013-7/"
 anyhow = "1.0.66"
 oidc4vp = { git = "https://github.com/spruceid/oidc4vp-rs" }
 openidconnect = { git = "https://github.com/ramosbugs/openidconnect-rs", branch = "main" }
-isomdl = { path = "../../isomdl" }
+isomdl = { git = "https://github.com/spruceid/isomdl.git" }
```

Comment:
does it work with `.git` at the end? Also, you should put the commit hash

Reply:
It did locally, I can chop it off so it matches the others above though.